src/test/java/com/microsoft/azure/proton/transport/proxy/impl/ProxyHandlerImplTest.java

```diff
@@ -76,11 +76,11 @@ public void testValidateProxyResponseOnFailure() {
 
     @Test
     public void testValidateProxyResponseOnInvalidResponse() {
-        final String invalidResponse = "HTTP/1.1 abc Connection Established\r\n" +
-                "HTTP/1.1 200 Connection Established\r\n" +
-                "FiddlerGateway: Direct\r\n" +
-                "StartTime: 13:08:21.574\r\n" +
-                "Connection: close\r\n\r\n";
+        final String invalidResponse = String.join("\r\n", "HTTP/1.1 abc Connection Established",
+            "HTTP/1.1 200 Connection Established",
+            "FiddlerGateway: Direct",
+            "StartTime: 13:08:21.574",
+            "Connection: close\r\n", "\r\n");

```

Comment:
Would this add one more `\r\n` than the original because we have a trailing `\r\n` in `close\r\n`?